recipes/OmniXAI/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "OmniXAI" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omnixai-{{ version }}.tar.gz
+  sha256: 2f8c9ed9ae3977b19f3fcd40a33a56b94fabc76ee63652c101dcaa39db40ba98
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<4.0
+    - pip
+  run:
+    - python >=3.7,<4.0
+    - numpy >=1.17
+    - pandas >=1.1.0
+    - scikit-learn >=0.24,<1.2
+    - scipy >=1.5.0
+    - scikit-image >=0.17.2
+    - matplotlib-base
+    - pillow <10.0
+    - lime
+    - shap >=0.40.0
+    - salib
+    - hnswlib >=0.5.0
+    - dill
+    - tqdm
+    - wheel
+    - packaging
+    - ipython
+    - tabulate
+    - statsmodels >=0.10.1
+
+test:
+  imports:
+    - docs
```

Comment:
This sounds wrong and it looks like it is installing its docs globally. Can you open an upstream bug and add an extra step that does `rm -rf ${SP_DIR}/docs`?

Reply:
When I remove the docs from the Tests - The import fails. 

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=627207&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823


`2022-12-12T14:12:51.1479352Z export PREFIX=/home/conda/staged-recipes/build_artifacts/omnixai_1670854274661/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol
2022-12-12T14:12:51.1483172Z export SRC_DIR=/home/conda/staged-recipes/build_artifacts/omnixai_1670854274661/test_tmp
2022-12-12T14:12:52.0443550Z import: 'omnixai'
2022-12-12T14:12:52.0444845Z Traceback (most recent call last):
2022-12-12T14:12:52.0446955Z   File "/home/conda/staged-recipes/build_artifacts/omnixai_1670854274661/test_tmp/run_test.py", line 2, in <module>
2022-12-12T14:12:52.0447892Z     import omnixai
2022-12-12T14:12:52.0448484Z ModuleNotFoundError: No module named 'omnixai'
2022-12-12T14:12:53.1578823Z Tests failed for omnixai-1.2.3-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
2022-12-12T14:12:53.1580894Z WARNING:conda_build.build:Tests failed for omnixai-1.2.3-pyhd8ed1ab_0.conda - moving package to /home/conda/staged-recipes/build_artifacts/broken
2022-12-12T14:12:53.1888425Z TESTS FAILED: omnixai-1.2.3-pyhd8ed1ab_0.conda`